package.json

```diff
@@ -36,8 +36,11 @@
     "release": "tasegir release --no-docs --no-compile --no-test --no-build",
     "prepublishOnly": "truffle compile && typechain --target=web3-v1 build/contracts/StorageManager.json && typechain --target=truffle-v4 --outDir ./types/truffle-v4-contracts build/contracts/StorageManager.json && typechain --target=truffle-v5 --outDir ./types/truffle-v5-contracts build/contracts/StorageManager.json && typechain --target=ethers-v4 build/contracts/StorageManager.json"
   },
-  "dependencies": {},
+  "dependencies": {
+    "@openzeppelin/test-environment": "^0.1.4"
```

Comment:
This should be `devDependency`, but I am not sure why you even need this dependency?

Reply:
Put it to `devDependency`. I'm using it in `stoppable.test.js`